### DISCUSSION TITLE
the best way to manage responsiveness ?

### DISCUSSION THREAD

**Discussion by pheninux** [charmbracelet/bubbletea#303] [Q&A] [answered]
hello, 
what is the best way to manage responsiveness with bubule tea, is there only this way to manage it , with :

we play on the width and the height we listen to the command tea.WindowSizeMsg ?
```
case tea.WindowSizeMsg:
		//1 - m.with, m.height = msg.Width, msg.Height
		//2 - m.with, m.height, _ = term.GetSize(int(os.Stdout.Fd()))
```

Cordially

**Comment by muesli** [comment, marked as answer]
Correct, every time the terminal gets resized, you will get a `tea.WindowSizeMsg` with the new width and height. There's no need to use `term.GetSize` manually.

**Reply by meowgorithm** [reply]
Also note that a `tea.WindowSizeMsg` will also be sent as the program is starting up, so you should also use it to the the initial terminal size.
